compair/static/modules/course/course-assignments-partial.html

```diff
@@ -67,7 +71,7 @@ <h2>{{filter}} <span ng-if="results.length != assignments.length">({{results.len
 
     <!-- Assignments List -->
     <div class="row each-assignment" ng-class="{'first-child': $first}"
-        ng-repeat="assignment in assignments | filter:assignmentFilter(filter) as results">
+        ng-repeat="assignment in assignments | filter:assignmentFilter(filter) | orderBy:assignmentSortOrder(sortOrder) as results">
```

Comment:
Please change
`orderBy:assignmentSortOrder(sortOrder)` to `orderBy:[assignmentSortOrder(sortOrder), '-created'] as results">`
(having created in descending order is a good secondary sort for all the different sort orders)